portafly/src/i18n/locales/en/product.yml

```diff
@@ -0,0 +1,39 @@
+---
+# ------------------ Product > Create product
+# Page details
+
+page_title: Create product #--- PROPOSED
+page_title_description: Page title to create a product
+body_title: Create a new product #--- PROPOSED
+body_title_description: Body title of the page to create a new product
+
+# Main area
+
+name: Name
+name_description: Field label to show/input the name of the product.
+
+system_name:
+  label: System name
+  label_description: Field label to show/input the system name of the product.
+  placeholder: product-system-name
+  placeholder_description: Placeholder to show an example for the product system name.
+  helper: Use letters, numbers, hyphens and underscores
+  helper_description: Helper text to explain the allowed characters for a system name.
+  popover: System names must be unique. You cannot edit a system name once the product is created.
```

Comment:
@porueesq as per conversation in chat, I believe we should be more descriptive here and explain that a System name is actually only unique within the context of the same type of objects. It is possible that we don't have to be so technical with wording, and maybe just say something like "There cannot be two Products sharing the same system name"? What do you think?

Reply:
@cntlsn Two suggestions for the first part of the inline help:
a. System names for products must be unique.
b. System names must be unique for all products.

Reply:
@porueesq Option A sounds good to me.

Reply:
Perfect. I'll add this update.